packages/fluentui/react-northstar/src/components/Tree/TreeTitle.tsx

```diff
@@ -120,7 +121,9 @@ export const TreeTitle: ComponentWithAs<'a', TreeTitleProps> & FluentComponentSt
     debugName: TreeTitle.displayName,
     actionHandlers: {
       performClick: e => {
-        e.preventDefault();
+        if (getCode(e) === SpacebarKey) {
```

Comment:
We can't remove `SpaceKey` from `treeTitleBehavior` because it should work for selectable `Tree` and it seems only makes sense to `preventDefault` for space to avoid page scroll. :( I don't like it but I am out of ideas

Reply:
Let's fix instead `shouldPreventDefaultOnKeyDown` to respect `a` tag & <kbd>Space</kbd> combination

Reply:
Done